avm/res/document-db/database-account/tests/e2e/gremlindb/main.test.bicep

```diff
@@ -20,6 +20,9 @@ param serviceShort string = 'dddagrm'
 @description('Optional. A token to inject into the name of each resource.')
 param namePrefix string = '#_namePrefix_#'
 
+// Pipeline is selecting random regions which dont support all cosmos features and have constraints when creating new cosmos
+var eastUsResourceLocation = 'eastus'
```

Comment:
In alignment with purview, I'd suggest to call it `enforcedLocation` if you'd agree (across all test files).